assemblies/karaf/features/src/main/resources/features.xml

```diff
@@ -4,7 +4,7 @@
     <feature>cxf</feature>
     <feature>cxf-http-async</feature>
     <feature>war</feature>
-    <bundle>wrap:mvn:com.clearspring.analytics/stream/${clearspring.analytics-stream.version}</bundle>
+    <bundle>wrap:mvn:com.clearspring.analytics/stream/${clearspring.analytics.stream.version}</bundle>
```

Comment:
I made that change to maintain consistency for properties names. The pattern I considered to give properties names is: {groupId or part of groupId}.{artifactId}.version